StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetFormFactory/PaymentSheetFormFactory+UPI.swift

```diff
@@ -14,8 +14,49 @@ import UIKit
 
 extension PaymentSheetFormFactory {
 
-    func makeUPI() -> FormElement {
-        return FormElement(autoSectioningElements: [makeUPIHeader(), makeVPAField()], theme: theme)
+    func makeUPI() -> PaymentMethodElement {
```

Comment:
In the form factory is it possible to do something like 

`makeUPI() + someReusableCodeForBDCC` rather than tying it to UPI? I haven't looked at other PR's so I'm not sure if other PMs can leverage it, but I would assume in the future we may want to reuse this code? Most of it looks agnostic to UPI.

Reply:
They are all slightly different, so abstracting the BDCC code is not straightforward, but I'll look into it more

Reply:
Rethinking about this a bit, if UPI happened to be added after BDCC, adding UPI would have been a bit more complex due to BDCC and would have been pretty easy to miss adding support for BDCC at all for UPI. Adding one-off payment methods would be more complex and potentially at risk of missing this functionality since it's not reusable for other one-off LPMs and they don't get it for free. Since UPI doesn't have any fields that would collide with billing address fields, it's still unclear why we need UPI-specific logic here. 
What is it about UPI that doesn't generalize well to other LPMs with respect to BDCC?

Reply:
Not being part of LUXE is exactly what makes it special, I can abstract the wrapper part to apply defaults (I'm working on it but haven't updated the PR yet), but adding the fields is not so easy to generalize, for example, for cards the fields are not just added at the end (same issue I had with LUXE, which was solved by adding placeholders), so I cannot just pass "additional elements" as a parameter, I can do that here but I don't think that solves the underlying issue, if in the future we add more one-off PMs the logic to add fields will also need to be added for them.
One thing about UPI though, is that according to @sfriedman-stripe it was never launched, so we can completely omit it if there is significant FUD around this.

Reply:
Got it, I think my main fud isn't necessarily related to UPI but more generally the effort to support BDCC on non-LUXE LPMs, so I don't think we need to omit UPI here.

Re: UPI not launched: It is currently in pilot, with beta hoping to launch in April. It's not clear if anyone has integrated with mobile.

Reply:
I agree with your FUD around non LUXE PMs, I'll try to abstract as much as possible to make it simpler, but I don't think there is a lot I can abstract

Reply:
I abstracted the defaults applying wrapper, I don't think I can abstract more than that, I think it looks more reasonable now